src/legacy/core_plugins/visualizations/public/embeddable/visualize_embeddable_factory.tsx

```diff
@@ -46,9 +28,8 @@ import {
   Container,
   EmbeddableOutput,
 } from '../../../../../plugins/embeddable/public';
-import { start as visualizations } from '../../../visualizations/public/np_ready/public/legacy';
-import { showNewVisModal } from '../visualize';
-import { SavedVisualizations } from '../visualize/np_ready/types';
+import { showNewVisModal } from '../../../kibana/public/visualize/np_ready/wizard/show_new_vis';
```

Comment:
This should also be exposed from the top level of  the `visualize` folder

Reply:
are you ok with leaving ti as is for now ? i tried top level import but that breaks some tests and i would like to keep this PR as small as possible.

in the follow up i will most likely move this to the visualizations plugin, as else we might end up with circular dependency (visualize_embeddable requires showNewVisModal, but kibana/visualize reqires a lot of things from visualization plugin)

Reply:
Ah right, I have a PR open that should fix this problem: https://github.com/elastic/kibana/pull/54124

Then it's fine for now, moving it over makes sense to me as this isn't exclusively "visualize" UI